layers/swoole/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION:1.7.14 AS ext
```

Comment:
Can you please add

```



ARG PHP_VERSION
ARG BREF_VERSION
FROM bref/build-php-$PHP_VERSION:$BREF_VERSION AS ext
```

See https://github.com/brefphp/extra-php-extensions/blob/1.0.0/layers/redis/Dockerfile